src/App.js

```diff
@@ -1,24 +1,26 @@
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
 import ProductDetails from "./pages/ProductDetails";
-import { ProductBox } from './components/ProductBox';
+import {ProductBox} from './components/ProductBox';
+
+import MainMenu from "./components/MainMenu";
 
 function App() {
-  return (
-    <div className="App">
-      <Router>
-        <Switch>
-          <Route path="/product/:productId" component={ProductDetails} />
-          <Route path="/"></Route>
-        </Switch>
-      </Router>
-      <ProductBox
-        id={1}
-        image="media/products/joust-bag.jpeg"
-        name="Joust Duffle Bag"
-        price="34€"
-      />
-    </div>
-  );
+    return (
+        <div className="App">
+            <MainMenu/>
```

Comment:
tak z ciekawości -> Czy <MainMenu/> nie musi być w Router'ze by linki działały w menu poprawnie, w sensie React way / bez przeładowania?

Reply:
Zmieniłem, ale nadal mam przeładowanie.

Reply:
By nie było przeładowania, trzeba chyba użyć Linka'a z react-router-dom'a?

Reply:
Oki, sprawdze wieczorem, chyba, że masz czas i ochotę to możesz naprawić :)

Reply:
też dopiero po południu mam czas jak coś

Reply:
podobno trzeba powiązać link z routera z tym z chakry https://chakra-ui.com/docs/navigation/link#usage-with-routing-library

Reply:
ma to sens :)

Reply:
Dzięki @majwro, działa :)